website/docs/faqs/Tests/custom-test-thresholds.md

```diff
@@ -10,5 +10,5 @@ As of `v0.20.0`, you can use the `error_if` and `warn_if` configs to set custom
 
```

Comment:
Same with this page, seems specific to data tests - may want to update to "Can I set **data** test failure thresholds?"